config/chips/L5x2.chip

```diff
@@ -0,0 +1,14 @@
+# Chip-ID file for STM32L5x2 device
+#
+dev_type STM32L5x2
+ref_manual_id 0438
+chip_id 0x472                // RM0438, pg. 2157
+flash_type L5_U5             // ???
```

Comment:
Please complete the missing comments according to the formatting scheme (see other chip-id-files), or remove them if there is no setting such as for the option-parametres.